### PR TITLE
Add endlessh collection

### PR DESCRIPTION
**Pull request by bamx23**
Hi everyone!

I've tried to find any kind of parsers/scenarios for the case when [Endlessh](https://github.com/skeeto/endlessh) is installed and `/var/log/auth.log` doesn't contain any information about 22-th port brute-forcing.
While Endlessh itself provides a kind of defense from such kind of attack, Crowdsec can't record them and ban attackers by IP (and improve security for everyone else in the crowd).

I've tried to implement a parser and a simple scenario for such a case. I've used the `sshd-logs` parser and the `ssh-slow-bf` scenario as a source of inspiration. I'm new here, so I would really appreciate any feedback.

### CODE REVIEW

File: collections/crowdsecurity/endlessh.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#the list of parsers it contains
+parsers:
+  - ./parsers/crowdsecurity/endlessh-logs.yaml
```
Comment (buixor) [inline collections/crowdsecurity/endlessh.yaml:3]: in the collection itself the items should referenced by their name, in this case `crowdsecurity/endlessh-logs` and `crowdsecurity/endlessh-bf`

File: collections/crowdsecurity/endlessh.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - crowdsecurity/dateparse-enrich
+scenarios:
+  - ./scenarios/crowdsecurity/endlessh-bf.yaml
+description: "endlessh support : logs parser and brute-force detection"
+author: bamx23
```
Comment (buixor) [inline collections/crowdsecurity/endlessh.yaml:8]: Unfortunately the name of the author must match the collection path for now (it is parsed by the hub app to generate the commands)
 - either you put `crowdsecurity` in the author name
 - or you keep `bamx23` but you add a `bamx23` directory for your parsers, scenarios and collection

**Review by buixor — CHANGES_REQUESTED** [review]
Some comments, but otherwise great job, thanks for the contribution :+1:

File: collections/crowdsecurity/endlessh.yaml (hunk 1 above)
Comment (bamx23) [reply collections/crowdsecurity/endlessh.yaml:3]: @buixor Should I also update it in the `.tests/endlessh-logs/config.yaml`?

File: collections/crowdsecurity/endlessh.yaml (hunk 2 above)
Comment (bamx23) [reply collections/crowdsecurity/endlessh.yaml]: `crowdsecurity` is fine, fixed

File: collections/crowdsecurity/endlessh.yaml (hunk 1 above)
Comment (buixor) [reply collections/crowdsecurity/endlessh.yaml]: sorry for the delay, looks good to me :)

**Comment by buixor** [comment]
LGTM, thanks :) wait for CI and merge

**Review by buixor — APPROVED** [review]

**Comment by bamx23** [comment]
@buixor sorry for the inconvenience. There likely was a race in test data, and I've likely solved it by increasing blackhole time. For the endlessh scenario, it should be fine to have it bigger.

**Comment by bamx23** [comment]
@buixor thanks for the review. I don't have access to merge, could you please do it?

**Comment by bamx23** [comment]
@buixor kindly remind about this PR

**Comment by buixor** [comment]
@bamx23 my bad, thanks for the reminder

**Pull request merged**